### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by jenky**
Require php extension gmp
Add autoload

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,11 +12,18 @@
 	],
 	"require":
 	{
-		"php": ">=5.4"
+		"php": ">=5.4",
+		"ext-gmp": "*",
```
Comment (xPaw) [inline composer.json]: Remove trailing comma here, otherwise it fails.

File: composer.json [hunk 2]
```diff
@@ -12,11 +12,18 @@
...
 	},
 	"require-dev":
 	{
 		"phpunit/phpunit": "4.7.x-dev",
 		"codeclimate/php-test-reporter": "dev-master"
+	},
+	"autoload":
+	{
+		"files": [
```
Comment (xPaw) [inline composer.json]: Mind adding a new line before `[`

**Comment by jenky** [comment]
Fixed

**Comment by xPaw** [comment]
Thanks!

**Pull request merged**